docs/quickstart/google-cloud-platform.md

```diff
@@ -0,0 +1,114 @@
+# Google Cloud Platform
+
+## Requirements
+
+This quick start assumes you have administrator access to your Google Cloud Platform instance. 
+
+## Setting Up your VM  
+
+If this is the first time you’re creating a VM on this project, you will need to enable the `Compute Engine API`. Click `enable` and wait for the service to finish connecting.
+
+This will pull up the `Create an Instance` page - name the instance something relevant to this project, following your naming convention of choice. In addition, select a region and zone that is relevant / close to your physical location. For this instance, we will use the base suggested image. 
+
+<img src="../images/quickstart/google-cloud-platform/gcp1.png">
+
+Under `Identity and API Access`, and click `Allow full access to all Cloud APIs`. Scroll down to `Firewall` and click `Allow HTTPS traffic` and `Allow HTTP traffic`.
+
+<img src="../images/quickstart/google-cloud-platform/gcp2.png">
+
+Scroll down to the bottom and click `Create` to create and deploy the VM.
+
+Congrats you’ve created your VM instance!
+
+## SSH-ing into the VM
+
+On the Compute Engine Dashboard, click on the VM for this project. Under `Details`, click `SSH` and select `Open in browser window`.
+
+<img src="../images/quickstart/google-cloud-platform/gcp3.png">
+
+This will give you a terminal to maneuver, manipulate the VM, and install Coder.
+
+## Install Coder
+
+In the terminal, run the following command 
+
+```sh
+curl -fsSL https://coder.com/install.sh | sh  
+```
+
+## Run Coder
+
+For this tutorial, we will run Coder as a System service. You can run Coder in [a multitude of different ways](https://coder.com/docs/coder-oss/latest/install).
```

Comment:
```suggestion
For this tutorial, we will run Coder as a `systemd` service. You can run Coder in [a multitude of ways](https://coder.com/docs/coder-oss/latest/install).
```

Reply:
We should either use "`systemd` service" or "system service". The proper noun "System" indicates that there is some piece of software called "System" to me. 

The advantage to specifying "systemd" is to separate us from other pid 1s such as launchd and upstart.

Reply:
This one was left unaddressed.